### PR TITLE
Add translations for views

### PR DESCRIPTION
**Pull request by kev-ac**
Relates to #324

### CODE REVIEW

File: pnpm-lock.yaml
Comment (cpvalente) [inline pnpm-lock.yaml:1]: What happened here? Would you mind checking out from v2 and re-running pnpm i?

File: apps/client/src/translation/TranslationProvider.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    getString,
+    setLanguage,
+  };
+
+  return <TranslationContext.Provider value={contextValue}>{children}</TranslationContext.Provider>;
+};
+
+export const useTranslation = () => {
+  const { getString, setLanguage } = useContext(TranslationContext);
+  return { getString, setLanguage };
```
Comment (cpvalente) [inline apps/client/src/translation/TranslationProvider.jsx:50]: I wonder if we could use something less generic like `getLocalisedString`
What do you think?

**Review by cpvalente — CHANGES_REQUESTED** [review]
This looks great to me. I made some small nitpicky comments. Mostly wondering about the pnpm-lock

Also, would you mind running the linter in the changed files? otherwise, it will pollute further commits
Context in routers sounds good. It shouldn't affect anything else

Would you mind implementing the other views or would you prefer me to?

File: pnpm-lock.yaml
Comment (kev-ac) [reply pnpm-lock.yaml:1]: Oops, this file shouldn't have been committed, I was too fast there. Will revert it.

File: apps/client/src/translation/TranslationProvider.jsx (hunk 1 above)
Comment (kev-ac) [reply apps/client/src/translation/TranslationProvider.jsx:50]: Yes, sounds like a good thing to do.

**Comment by kev-ac** [comment]
> Also, would you mind running the linter in the changed files? otherwise, it will pollute further commits

Eslint runs in my IDE automagically when I save a file, so there shouldn't be any issues here. Also a manual run doesn't give any recommendations. 

> Would you mind implementing the other views or would you prefer me to?

I can do it, but I could take some time. Doing this in some free time in-between other things, surely how you also do it :-) 
So feel free if you have some spare time.

**Comment by cpvalente** [comment]
I am working on some other small fixes and can do that as well. 

Do you think we can merge this? I am happy to approve of you remove the draft flag

**Pull request merged**